docs/api-reference/euler.md

```diff
@@ -59,6 +59,23 @@ rotation order in all notations
  * Number|Number[], Number, Number, Number
 
 
+### fromRollPitchYaw
+
+Common ZYX rotation order
+
+`euler.fromRollPitchYaw(roll, pitch, yaw)`
+
+
+### fromRotationMatrix
+
+`euler.fromRotationMatrix(m, order = Euler.DefaultOrder)`
+
+
+### fromQuaternion
+
+`euler.fromQuaternion(q, order)`
```

Comment:
These should be static methods?

Reply:
> These should be static methods?

Interesting you should say that. I did exactly that as my first iteration when creating the library, but abandoned it for a couple of reasons. 
* While conceptually more elegant, the resulting user code did not feel simpler. 
* Also, I thought that having math.gl highly API compatible with THREE.js math library would be cool and they do it this way (math.gl runs and passes a big subset of the THREE.js math library test suite).
* Since the default constructor still has to be run, making these "overloaded" constructors static does not actually improve performance.

I also though about supporting both static and member versions of the `from*` functions, but that would either:
    * require duplicating method stubs (not good for bundle size, I feel that a good math library should have a small footprint)
    * create dynamic class creation code (add methods to prototype and class from a table) but that has side effects and is incompatible with tree-shaking.